core/cas-server-core/src/test/java/org/apereo/cas/BaseCasCoreTests.java

```diff
@@ -124,7 +125,8 @@ public ExpirationPolicy buildTicketExpirationPolicy() {
         CasMultifactorAuthenticationWebflowConfiguration.class,
         CasPersonDirectoryTestConfiguration.class,
         CasCoreValidationConfiguration.class,
-        CasCoreConfiguration.class
+        CasCoreConfiguration.class,
+        CasAuthenticationDelegationTestConfiguration.class
```

Comment:
Same as above: the module here is a "core" module and pac4j belongs to the support category.

Reply:
Copy that